homeassistant/helpers/config_entry_oauth2_flow.py

```diff
@@ -283,7 +287,7 @@ async def async_step_auth(
             return self.async_external_step_done(next_step_id=next_step)
 
         try:
-            async with async_timeout.timeout(10):
+            async with async_timeout.timeout(OAUTH_AUTHORIZE_URL_TIMEOUT_SEC):
                 url = await self.async_generate_authorize_url()
         except asyncio.TimeoutError:
             return self.async_abort(reason="authorize_url_timeout")
```

Comment:
Should we log an error here too, similar to the token creation timeout case?

Reply:
Done